charts/kubernetes-agent/values.yaml

```diff
@@ -339,6 +339,12 @@ persistence:
         pullPolicy: IfNotPresent
         tag: "0.2.0"
 
+ # Used to enable the Kubernetes Monitor
+# Do not use this, as it is not supported at the moment
+# @ignored
+kubernetes-monitor-chart:
```

Comment:
could just be `kubernetesMonitor`?
The fact it's a chart is kinda irrelevant for the values?

Reply:
When setting values, the sub-charts name is the root of the values for that particular chart. This allows us to expose (in future) the documentation for the subchart in our readme. If we use `kubernetesMonitor`, we'd need to name the chart `kubernetesMonitor`, which clashes with the _actual_ image 😅

Reply:
gotcha 👍